pkg/handler/community/interface.go

```diff
@@ -35,4 +35,7 @@ type IHandler interface {
 	InitAdSubmission(c *gin.Context)
 	DeleteAdById(c *gin.Context)
 	UpdateAdById(c *gin.Context)
+
```

Comment:
i think that we do not need this line break, if want to group methods, can add comment above them.